packages/hooks/src/__tests__/useCancelling.ts

```diff
@@ -34,6 +34,10 @@ describe('useCancelling()', () => {
       .delay(2000)
       .reply(200, payload2);
   });
+  afterAll(() => {
+    jest.useRealTimers();
+    nock.cleanAll();
+  });
```

Comment:
should this be after _all_, or after _each_?

Reply:
It was before all for the setup. Does jest interleave tests?

Reply:
given that timers and nock are per-process, using "all" means that tests in the same process (which might or might not be as granular as "same file") could be depending on the timer/nock behavior from other tests to pass. Using "each" ensures every test runs in isolation from the others in this respect.